### ISSUE TITLE
Publishing with https://update-cardano-mainnet.iohk.io

### ISSUE THREAD

**Issue by Fell-x27** [input-output-hk/cardano-addresses#222]
Many IOG products are listed with https://update-cardano-mainnet.iohk.io, so we have a handy software registry and downloading pipeline. It's very useful in terms of auto-deploying systems. 

Maybe cardano-addresses should be published the same way?

**Comment by abailly** [comment]
@Fell-x27 I just discovered the existence of this, how is it used?

**Comment by Fell-x27** [comment]
> @Fell-x27 I just discovered the existence of this, how is it used?

You can use it to fetch a list of releases and show it in your application, for example, or for automating updates in scripts, etc.

**Comment by abailly** [comment]
Is this some standard mechanism? Pardon my ignorance :)

**Comment by abailly** [comment]
Also, please note cardano-addresses is _not_ an IOG product ;)

**Comment by Fell-x27** [comment]
> Is this some standard mechanism?
Nope;


> please note cardano-addresses is _not_ an IOG product ;)
No, it actually is. If you check, the link is related to an IOHK repository. Yes, it is a project of "Intersect," but that doesn't mean IOG is not the maintainer — like the Node, for example.

**Comment by abailly** [comment]
> No, it actually is. If you check, the link is related to an IOHK repository. Yes, it is a project of "Intersect," but that doesn't mean IOG is not the maintainer — like the Node, for example.

No, it's not: The core maintainers, ie. the people actually writing code, are working at the CF. `CODEOWNERS` show that @paweljakubas  (CF employee) is owning the core Haskell part while @Crypto2099 (independent/Intersect) is owning the rest.

**Comment by abailly** [comment]
To get back to the initial request: I would rather make cardano-addresses installable using a standard mechanism, like an `apt` or `brew` package.
